scio-elasticsearch/src/main/java/org/apache/beam/sdk/io/elasticsearch/ThrowingConsumer.java

```diff
@@ -18,8 +18,9 @@
 package org.apache.beam.sdk.io.elasticsearch;
 
 import java.io.Serializable;
-import java.util.function.Consumer;
 
-public interface SerializableConsumer<T> extends Consumer<T>, Serializable {
+@FunctionalInterface
+public interface ThrowingConsumer<T> extends Serializable {
 
```

Comment:
@lmayumi good point ... this error handler is just for the bulk exception. I'll keep ThrowableConsumer generic enough but simpler ... let me know about the following fixup.